config/Common.enterprise.xcconfig

```diff
@@ -0,0 +1,3 @@
+#include "Common.xcconfig"
+
+DEVELOPMENT_TEAM = 99KV9Z6BKV
```

Comment:
I decided to create a single "enterprise" file instead of two dedicated "internal" and "alpha" because for the moment the two configuration use the same value. If/when we'll need build configuration-specific build settings, we can add dedicated files for them.